expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsPolygons.kt

```diff
@@ -0,0 +1,30 @@
+package expo.modules.maps
+
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.Polygon
+import com.google.android.gms.maps.model.PolygonOptions
+
+class GoogleMapsPolygons(map: GoogleMap): Polygons {
+    private val polygons = mutableListOf<Polygon>()
+    private var googleMap: GoogleMap = map
+
+    override fun setMarkers(polygonObjects: Array<PolygonObject>) {
```

Comment:
You forgot to rename this function, I presume.
```suggestion
    override fun setPolygons(polygonObjects: Array<PolygonObject>) {
```